### DISCUSSION TITLE
Authentication Problem

### DISCUSSION THREAD

**Discussion by Korte-IT** [Lissy93/dashy#1132] [Help and Support]
When i update the image of dashy beyond 2.0.8, for example the latest version, my authentication stops working and dashy doesnt ask for a login. If i change back to Version 2.0.8 everything is working fine. Did something change, that i didnt notice?
Thanks for your attention. I’m looking forward to your reply.

**Comment by mb0e** [comment]
Hello, I am experiencing a similar problem. If I delete the /login path it will access the dashboard even though I don't have guests allowed. If I were to make a change in the workspace it will then ask for login.

**Comment by mspeculatrix** [comment]
I'm having a similar problem. In the config I have explicitly disabled guest login and set up two users. I have cleared cookies, local storage and session storage. The config shows as valid and the logs show the Dashy container booting successfully. Yet when I connect, I'm granted access with no login. But I'm connected as a guest, and can't globally save any changes. There's no logout button and going to /login just bumps me to the dashboard. I'm using 2.1.1.

**Comment by kt-alt** [comment]
Experiencing the same behavior here with 2.1.1.  However if I change version to 2.0.8 in the docker-compose.yml, it works.

Actually with a fresh instance, if you try to add a user/hash in the UI, it won't even allow you to save to disk.  I had to add it in the conf.yml manually and restart the container.

Seems like a bug?  Or am I don't something wrong?

**Reply by artemdanielov** [reply]
exactly same!

**Comment by mrkhachaturov** [comment]
Same bug!

**Reply by mrkhachaturov** [reply]
Issue on 2.1.1 still remains.

Found a way to force to login page. Just enter any path after your dashy domain , /ajaaj for example, it will return dashy error page not found. There will be button - back home. When pressing the button it redirects to login page.
